packages/react-native-tracer-provider/android/gradle.properties

```diff
@@ -1,5 +1,9 @@
-TracerProvider_kotlinVersion=1.7.0
-TracerProvider_minSdkVersion=21
-TracerProvider_targetSdkVersion=31
-TracerProvider_compileSdkVersion=31
+android.useAndroidX=true
+
+TracerProvider_packageJsonPath=..
```

Comment:
for consistency with `packages/react-native-otlp/android/gradle.properties` might be better to prefix these with `RNEmbraceTracerProvider_`